ui/admin/tests/helpers/selectors.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+export const SAVE_BTN = '[type=submit]';
+export const CANCEL_BTN = '.rose-form-actions [type=button]';
+export const EDIT_BTN = '.rose-form-actions [type=button]';
+
+export const FIELD_NAME = '[name=name]';
+export const FIELD_NAME_VALUE = 'Random name';
+
+export const TABLE_ROW_RESOURCE_LINK = 'tbody tr:first-child a';
+
+export const RESOURCE_NOT_FOUND_SUBTITLE = '.rose-message-subtitle';
+export const RESOURCE_NOT_FOUND_VALUE = 'Error 404';
+
```

Comment:
I added spacing where it seemed logical. I think this can help with readability compared to a large block with no breaks. Let me know if you have thoughts on this.